pkg/kubelet/apis/well_known_labels.go

```diff
@@ -27,7 +27,10 @@ const (
 	// The OS/Arch labels are promoted to GA in 1.14. kubelet applies both beta
 	// and GA labels to ensure backward compatibility.
 	// TODO: stop applying the beta OS/Arch labels in Kubernetes 1.18.
-	LabelOS   = "beta.kubernetes.io/os"
+
+	// LabelOS is a label to indicate the operating system of the node.
```

Comment:
Keep the comments about backward compatibility and the release they will no longer be applied associated with the variables

Reply:
Thank you very much! @liggitt 
I have updated the code.